### DISCUSSION TITLE
[求助]css过渡结束时文本垂直水平居中作用于网页

### DISCUSSION THREAD

**Discussion by shidijun** [share121/inter-knot#5285] [绝区零]
个人想要的效果是文本作用于方块（即div）可是没有将鼠标悬浮在方块上（即没有发生过渡效果）时文本的垂直水平居中作用于网页 实在不知道怎么写
<b>代码如图</b>
<img src="https://github.com/user-attachments/assets/1e7cbb16-1a15-4062-ad04-48c32a64cf33">
<b>效果如下</b>
<img src="https://github.com/user-attachments/assets/aa86ce1b-2cfa-41c6-b9c1-696bf271ffde">
<img src="https://github.com/user-attachments/assets/addf9a94-4cbf-4fdd-b038-376669bb043a">
或者访问我的网页
https://ovoshidijunovo.rth1.xyz/test.html

**Comment by Hyirt** [comment]
`<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>史蒂君</title>
			<link href="https://s21.ax1x.com/2024/06/01/pkGpbaF.jpg" rel="shortcut icon">
		<style>
			div {
				width: 100px;
				height: 100px;
				background: yellow;
				transition: width 2s, height 2s, transform 2s;
				position: relative;
			}
			div:hover {
				width: 200px;
				height: 200px;
				transform: rotate(360deg);
			}
			div>p {
                margin: 0;
  			    position: absolute;
 			    top: 50%;
   			    left: 50%;
  			    -ms-transform: translate(-50%, -50%);
  			    transform: translate(-50%, -50%);
			}
		</style>
	</head>
	<body>
	<div>
		<p>LOVE</p>
	</div>
	</body>
</html>
`

**Reply by shidijun** [reply]
谢谢<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >
